src/libraries/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -12,6 +12,8 @@ namespace System.IO
 {
     public class EnumerationOptions
     {
+        private int _maxRecursionDepth = DefaultMaxRecursionDepth;
```

Comment:
For consistency can you please set this field in the ctor.

Reply:
Ah, it is already in ctor :-)

Dup initialization removed.